package.json

```diff
@@ -3,8 +3,8 @@
   "version": "3.2.0",
   "main": "server.js",
   "scripts": {
-    "dev": "nodemon --watch app --watch bootstrap --watch config --watch .env -x node server.js",
-    "start": "node server.js",
+    "serve:dev": "nodemon --watch app --watch bootstrap --watch config --watch .env -x node server.js",
```

Comment:
It will be nice to have fallbacks on `dev` and `start` with a warning. For example:

```
{
   "dev": "echo \"\\033[31mWARN: 'dev' will be depreciated soon. Instead use 'npm run serve:dev'\" && npm run serve:dev"
}
```